_posts/2022-09-29-r-text/r-text.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+title: "The text-package"
+description: |
+  The [text](http://r-text.org/)-package attempts to provide user-friendly access and pipelines to HuggingFace's transformer language models in R.
```

Comment:
Unfortunately, you cannot use this kind of formatting in the abstract (see preview for how it does not display correctly).
I'd just go with 
"The text package" (no hyphen)

Reply:
it's been removed.